packages/calypso-products/src/features-list.tsx

```diff
@@ -2630,6 +2632,22 @@ const FEATURES_LIST: FeatureList = {
 	},
 	/* END: Sensei Features */
 
+	[ FEATURE_BIG_SKY_WEBSITE_BUILDER ]: {
+		getSlug: () => FEATURE_BIG_SKY_WEBSITE_BUILDER,
+		getTitle: () =>
+			i18n.translate( '{{strong}}Unlimited AI Website Builder edits{{/strong}}', {
+				components: {
+					strong: <strong />,
+				},
+			} ),
+		getDescription: () => i18n.translate( 'Build your site with our AI Website Builder.' ),
+	},
+	[ FEATURE_BIG_SKY_WEBSITE_BUILDER_CHECKOUT ]: {
```

Comment:
The copy on the plans page should be bold, so we create a non bold one for the checkout page, maybe there is a better way to do this?